static_report/src/components/shared/Sidebar.tsx

```diff
@@ -0,0 +1,177 @@
+import {
+  Box,
+  Button,
+  Divider,
+  Flex,
+  Icon,
+  Text,
+  Modal,
+  ModalHeader,
+  ModalOverlay,
+  ModalContent,
+  ModalCloseButton,
+  ModalBody,
+  ModalFooter,
+  Textarea,
+  Heading,
+  UnorderedList,
+  ListItem,
+  useDisclosure,
+  useToast,
+} from '@chakra-ui/react';
+import { FiDatabase, FiMessageSquare } from 'react-icons/fi';
+import { useState } from 'react';
+
+type Feedback = {
+  user_id: string;
+  message: string;
+  version?: string;
+  score?: number;
+  page?: string;
+  misc?: string;
+};
+
+async function sendFeedback(data: Feedback): Promise<boolean> {
+  try {
+    await fetch(
+      `https://api.aws.piperider.io/v1/feedback?token=371aa67a-a6eb-44cc-9d02-4a05810f24ac`,
```

Comment:
Do you know if there's a TTL expiration on the `token`?
* Just making sure this doesn't stop working in 30 days or sth
* Also, just double-checking, is this sensitive env variable?

Reply:
I discussed this with @kentwelcome, we put the token directly (I think it will not be expired) because putting in the env cannot create a feedback message.

I think it can be improved without exposing the token parameter.

Reply:
Currently, this token is configured in our feedback API backend. It will not expire.
Ideally, it will be replaced once we implement the SaaS version.